.github/workflows/build.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       matrix:
         node: ["16.0.0", "14.16.1", "12.22.1", "10.24.1"]
-        ruby: ["3.0.1", "2.7.3", "2.6.7", "2.5.9"]
+        ruby: ["3.0.3", "3.0.1", "2.7.3", "2.6.7", "2.5.9"]
```

Comment:
Do you still need "3.0.1"?

Reply:
@texpert yep. There might be another project that uses devcontainer besides mine current. They might still want to continue using 3.0.1 until they fix it in Gemfile or other Dockerfiles.